packages/parent-bridge/contracts/parent-bridge/src/main.rs

```diff
@@ -104,17 +121,36 @@ impl StateTransition {
                 return Err(Error::EmptyValidatorList);
             };
             let state_id: Bytes = get_state_id()?;
-            debug!("validators: {:?}", validators);
             return Ok(StateTransition::DeployBridge {
                 validators: validators,
                 id: state_id,
-            })
+            });
         }
 
         let mut wit_buf: [u8; 1] = [0];
         load_witness(&mut wit_buf, 0, 0, Source::Input)?;
 
         match wit_buf[0] {
+            1 => {
+                let bridge_cap_before = load_cell_capacity(0, Source::Input)?;
+                let bridge_cap_after = load_cell_capacity(0, Source::Output)?;
+                let total_deposit_capacity = QueryIter::new(load_cell_lock, Source::Input)
+                    .zip(QueryIter::new(load_cell_capacity, Source::Input))
+                    .filter(|(script, _)| {
+                        *script.code_hash().raw_data() == DEPOSIT_LOCK_CODE_HASH[..]
+                    })
+                    .map(|(_, cap)| cap)
+                    .sum();
+                let data_before = load_cell_data(0, Source::Input)?;
+                let data_after = load_cell_data(0, Source::Output)?;
+                Ok(Self::CollectDeposits {
+                    total: total_deposit_capacity,
```

Comment:
Informational question? Are these values attributed to the parent bridge cell? Is there a way to read them from cell data or we can just read the capacity of bridge cell only?

Reply:
Not sure I understand the question.